core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -221,7 +226,20 @@ public void drawPlace(int x, int y, int rotation, boolean valid){
         Draw.color(Pal.placing);
         Drawf.circles(x * tilesize + offset(), y * tilesize + offset(), laserRange * tilesize);
 
-        getPotentialLinks(tile, other -> Drawf.square(other.drawx(), other.drawy(), other.block().size * tilesize / 2f + 2f, Pal.place));
+        getPotentialLinks(tile, other -> {
+            Drawf.square(other.drawx(), other.drawy(), other.block().size * tilesize / 2f + 2f, Pal.place);
+
+            insulators(tile, other).each(cause -> {
```

Comment:
Can be replaced with a direct consumer as mentioned previously.

Reply:
Not entirely sure what you mean by this, but i may have accidentally solved it 😗